### PR TITLE
add tensorflow-addons

### PR DESCRIPTION
**Pull request by Fissium**
add tensorflow-addons=0.12.0

### CODE REVIEW

**Comment by marcalph** [comment]
Hi @Fissium,
IMO in `test_installs.py` it should be tensorflow_addons (and not a hyphen)

**Comment by marcalph** [comment]
OK scratch that, it looks like conda doesn't have a great support for tf addons.. Maybe add a different channel?

**Comment by isms** [comment]
@cszc Request your review on this one

File: runtime/py.yml [hunk 1]
```diff
@@ -22,4 +23,4 @@ dependencies:
   - xgboost=1.2.0=py38h950e882_0
   - cloudpickle=1.3.0
   - tsfresh=0.17.0
-  - pytorch-lightning=1.1.2
```
Comment (cszc) [inline runtime/py.yml:25]: @Fissium Can you add a new line here for `tensorflow-addons` instead of replacing the line for `pytorch-lightning`?

**Review by cszc — CHANGES_REQUESTED** [review]

**Comment by cszc** [comment]
@Fissium do you still need this merged?

**Comment by Fissium** [comment]
Hi @cszc,
Yes, I still want to add the package to the runtime, but I don't quite understand what needs to be done. 
Can you explain to me?

**Comment by cszc** [comment]
@Fissium no worries! I made the change directly. You should be all set now.

**Comment by Fissium** [comment]
@cszc Thank you very much)

**Pull request merged**